package.json

```diff
@@ -44,7 +44,8 @@
   },
   "pnpm": {
     "overrides": {
-      "semver": "^7.5.2"
+      "semver": "^7.5.2",
+      "typescript": "^5.6.2"
```

Comment:
@DanRod1999, version override shouldn't be required to use build-tools extract-api.

https://github.com/iTwin/itwinjs-core/pull/7552 should fix this

Reply:
[5.0.0-dev.49](https://www.npmjs.com/package/@itwin/build-tools/v/5.0.0-dev.49) is out. You can try upgrading to it and removing TypeScript version override.

Reply:
Thanks @GytisCepk.